rollup.config.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import path from 'path'
+import resolve from 'rollup-plugin-node-resolve'
+import babel from 'rollup-plugin-babel'
+import replace from 'rollup-plugin-replace'
+import commonjs from 'rollup-plugin-commonjs'
+import { uglify } from 'rollup-plugin-uglify'
+import pkg from './package.json'
+
+function getConfig() {
+  const name = 'styledSystem'
+  const buildName = 'styled-system'
+
+  const SOURCE_DIR = path.resolve(__dirname, 'src')
+  const DIST_DIR = path.resolve(__dirname, 'dist')
+
+  const baseConfig = {
+    input: `${SOURCE_DIR}/index.js`,
+    plugins: [
+      babel({
+        runtimeHelpers: true,
+        exclude: 'node_modules/**',
+        configFile: path.join(__dirname, 'babel.config.js'),
+      }),
+    ],
+  }
+
+  const esConfig = Object.assign({}, baseConfig, {
+    output: {
+      file: `${DIST_DIR}/${buildName}.es.js`,
+      format: 'es',
+    },
+    external: [
+      ...Object.keys(pkg.peerDependencies || {}),
+      ...Object.keys(pkg.dependencies || {}),
+    ],
```

Comment:
`id => !id.startsWith('.') && !id.startsWith('/')` is more universal. It will treat as external `@babel/runtime/...`. Make sure your config does this.